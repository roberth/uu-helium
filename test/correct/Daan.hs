module Daan where

main :: Int
main = let f x = f (const x 1) in f 0
   