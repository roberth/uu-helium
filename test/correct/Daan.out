Compiling correct/Daan.hs
Compilation successful
KNOWN BUG: lvmrun does not cleanly detect out of memory here.

